riiif.gemspec

```diff
@@ -20,7 +20,8 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'railties', '>= 4.2', '<7'
   spec.add_dependency 'deprecation', '>= 1.0.0'
-  spec.add_dependency 'iiif-image-api', '~> 0.1.0'
+  spec.add_dependency 'iiif-image-api', '>= 0.2.0'
```

Comment:
Does this need to be `>= 0.2.0` or could `>= 0.1.0` also work?

Reply:
it would have to be `> 0.1.0`, since that's the version we're trying to get off of.  but since there's no other releases in `0.1.x`, i don't see what difference it would make.

Reply:
or do you mean to allow other riiif applications to continue using `0.1.0`?

Reply:
Yep that is what I mean. Just loosening up the requirement so that `>= 0.1.0` could allow other riiif applications or consumers to use other versions of that gem. (essentially so updaters could update, but those who don't want to don't have to. If that is the only Rails 6 adjustment for Riiif.. I _think_ `>= 0.1.0` should work.

Reply:
done